### ISSUE TITLE
Rule proposal: Prefer assert() or assert.ok()

### ISSUE THREAD

**Issue by jonahsnider** [sindresorhus/eslint-plugin-unicorn#1806]
### Description

Can be useful for enforcing a consistent coding style when using `node:assert`.

### Fail

When `assert.ok()` should be preferred:
```js
assert(condition);
```

When `assert()` should be preferred:
```js
assert.ok(condition);
```

Generally just a bad idea (but this does actually work):
```js
assert.ok.ok.ok.ok.ok.ok.ok.ok.ok.ok.ok.ok(condition);
```

### Pass

When `assert.ok()` should be preferred:
```js
assert.ok(condition);
```

When `assert()` should be preferred:
```js
assert(condition);
```

### Additional Info

[Relevant docs](https://nodejs.org/dist/latest-v16.x/docs/api/assert.html)

**Comment by sindresorhus** [comment]
Can you suggest a name for the rule?

**Comment by jonahsnider** [comment]
`assert-ok-style`?

**Comment by sindresorhus** [comment]
That would be fine. However, I think I prefer `consistent-assert`. That also leaves it open to other assert consistency linting.

**Comment by sindresorhus** [comment]
Accepted
